solidity/contracts/BancorNetwork.sol

```diff
@@ -10,8 +10,10 @@ import './utility/SafeMath.sol';
 import './utility/interfaces/IContractRegistry.sol';
 import './utility/interfaces/IContractFeatures.sol';
 import './utility/interfaces/IWhitelist.sol';
+import './utility/interfaces/ITokenWhitelist.sol';
```

Comment:
what's the logic behind that name? why whitelist?

Reply:
Not the most intuitive name, just didn't want to spend too long thinking about the name... we should think of a more explanatory name